scrapers/EliteBabes.yml

```
@@ -0,0 +1,50 @@
+name: EliteBabes
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.elitebabes.com/?s={}
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - www.elitebabes.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //article[@id='content']/ul[@class='gallery-a b d']/li/a[contains(@href,"model/")]/span/text()
+      URL:
+        selector: //article[@id='content']/ul[@class='gallery-a b d']/li/a[contains(@href,"model/")]/@href
```

```suggestion
        selector: //article[@id='content']/ul[1]//a/@href
```

Similar strategy to the above here

---

Same

---

```suggestion
        selector: //article[@id='content']/*[following-sibling::h2]//a/@href
```

Same new suggestion from above

---

```suggestion
        selector: //ul//a[contains(@href,"model/")]/@href
```

Super short version that leaves the query in and strips out the unneeded parts of the path